core/commands/nimcheck.py

```diff
@@ -114,7 +114,7 @@ def run_nimcheck(file_path, callback, verbosity, disabled_hints, extra_args):
         yield callback(None)
 
     entries = parse_nimcheck(stdout)
-    sublime.status_message('Nim Check Finished.')
+    sublime.status_message('run_nimcheck:Nim Check Finished.')
```

Comment:
that's because `Nim Check Finished.` is now ambiguous (2 identical entries), so I want to distinguish it in logs